.gitignore

```diff
@@ -14,3 +14,7 @@ build/astra_analytical/build/
 build/astra_analytical/result/
 build/astra_garnet/build/
 build/astra_garnet/result/
+
+# Generated Chakra ET files
+*.eg
```

Comment:
Thanks for adding this. We don't need *.eg anymore.

Reply:
Basically I put this because the Chakra submodule (commit 012243a7 specifically) is generating *.eg files, though the generator itself is renamed as `et_generator`. We should of course update the Chakra submodule version later; but if adding *.eg is not losing anything, we may consider keeping it (as previous versions of Chakra had used this extension).